### PR TITLE
SPAC2020

### PR DESCRIPTION
**Pull request by flora8984461**

### CODE REVIEW

**Comment by webmaster-ieeecarleton** [comment]
In the future, please split these in to multiple logical commits to help me review the content faster.

EDIT: I will actually ask that the commit description is changed to something along the lines of "Front-end overhaul for the 2020 year"

**Comment by dynamic11** [comment]
I believe those are very old commits. I don’t know how they showed up in the pull request.
-I’m the old webmaster

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-ieeespac.ca
```
Comment (webmaster-ieeecarleton) [inline CNAME]: The CNAME record is required for github's DNS to reflect that the *.github.io are alias domains. Please remove this change.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+language:      node_js
```
Comment (webmaster-ieeecarleton) [inline .travis.yml]: I don't believe we currently have travis enabled. Please make a new pull request for travis specifically, if you'd like to add CI.

**Comment by flora8984461** [comment]
> In the future, please split these in to multiple logical commits to help me review the content faster.
> 
> EDIT: I will actually ask that the commit description is changed to something along the lines of "Front-end overhaul for the 2020 year"

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+node_modules
+bower_components
+.idea/workspace.xml
```
Comment (webmaster-ieeecarleton) [inline .gitignore]: This specific include should be part of the local gitignore. Please remove line 3.

**Comment by webmaster-ieeecarleton** [comment]
Reopened.

**Comment by webmaster-ieeecarleton** [comment]
@dynamic11 No, the file changes are fine, but they are only included in one commit. This makes reviewing piece-by-piece much trickier. 

Most of the deletions are fine, but may warrant a re-write of history after this year to remove lots of the old binaries.

File: LICENSE.txt [hunk 4]
```diff
@@ -1,10 +0,0 @@
-
-
-
-The MIT License (MIT) Copyright (c) 2017 markups.io
```
Comment (webmaster-ieeecarleton) [inline LICENSE.txt:4]: Why was the license removed?

(Please update the copyright header!)

File: README.md [hunk 5]
```diff
@@ -1,2 +0,0 @@
-# SPAC2018Template
```
Comment (webmaster-ieeecarleton) [inline README.md:1]: Could you please update the readme instead of deleting it?

**Comment by flora8984461** [comment]
@webmaster-ieeecarleton 
I apologize, I should've made sure I specified clearly that it was front-end oriented. I'll make sure I'll specify next time when I have the back-end for the ticket form ready.

**Review by webmaster-ieeecarleton — COMMENTED** [review]
This is a review of the first 97 changes. I will review the bigger additions piecewise.

**Comment by rafiddewan** [comment]
@flora8984461, maybe create a feature branch and the other two branches I mentioned to you earlier. But maybe work on this repository? Here is what I outlined earlier.  @webmaster-ieeecarleton what do you propose?
Git Version Control

File: dummy.txt [hunk 6]
```diff
@@ -1 +0,0 @@
-dummy.txt
```
Comment (webmaster-ieeecarleton) [inline dummy.txt:1]: Can this file be removed entirely? I'm not sure why it's here.

**Review by webmaster-ieeecarleton — COMMENTED** [review]
CSS-fonts: looks good except the dummy text file

**Comment by webmaster-ieeecarleton** [comment]
@rafiddewan mind pulling that discussion into an issue? its not something that should be done in the next 12 hours.

**Review by webmaster-ieeecarleton — COMMENTED** [review]
For the images: any image > 100 KB should be checked if further compression is possible. Images > 1 MB are unacceptable. The loading time for some of these images will be poor. I'll accept them due to the deadline, but these need to be addressed in another update after this is merged ASAP.

File: CREDITS.txt [hunk 7]
```diff
@@ -1,56 +1,56 @@
---------------------------------------------
-Resources Credits : 
---------------------------------------------
-
-Bootstrap :
-	Website URL : http://getbootstrap.com/
```
Comment (webmaster-ieeecarleton) [inline CREDITS.txt:6]: These credits were correctly removed in the original commit. Not a blocker.

File: vendor/fontawesome-free/css/v4-shims.css [hunk 8]
```diff
@@ -0,0 +1,2166 @@
+/*!
```
Comment (webmaster-ieeecarleton) [inline vendor/fontawesome-free/css/v4-shims.css]: This should probably be minified.

File: vendor/fontawesome-free/css/fontawesome.css [hunk 9]
```diff
@@ -0,0 +1,4359 @@
+/*!
```
Comment (webmaster-ieeecarleton) [inline vendor/fontawesome-free/css/fontawesome.css]: This should be minified in a seperate pull request, not right now.

**Comment by webmaster-ieeecarleton** [comment]
Looks good overall. I will test tomorrow morning and merge if all is well after I get some sleep :zzz:

File: dummy.txt (hunk 6 above)
Comment (flora8984461) [reply dummy.txt:1]: I am sorry I am not sure why this file shows up, but it is not in my repo.

**Comment by flora8984461** [comment]
@webmaster-ieeecarleton 
Sorry missed the big pics, already compressed, every pic should be smaller than 1M now

**Pull request merged**